skimage/data/_fetchers.py

```diff
@@ -96,9 +96,36 @@ def _create_image_fetcher():
         skimage_version_for_pooch = __version__.replace('.dev', '+')
 
     if '+' in skimage_version_for_pooch:
-        url = "https://github.com/scikit-image/scikit-image/raw/" "{version}/skimage/"
+        url = "https://github.com/scikit-image/scikit-image/raw/{version}/skimage/"
     else:
-        url = "https://github.com/scikit-image/scikit-image/raw/" "v{version}/skimage/"
+        url = "https://github.com/scikit-image/scikit-image/raw/v{version}/skimage/"
+
+    is_jupyterlite = False
+    try:
+        import pyodide_kernel  # noqa: F401
+
+        is_jupyterlite = True
+    except (ModuleNotFoundError, ImportError):
+        pass
+
+    # This code path modifies the registry_urls dictionary to route all
+    # GitLab resources through a CORS proxy (cdn.statically.io), making
```

Comment:
Is there a problem with always routing via CDN, instead of just for pyodide?

Reply:
My opinion is that we should avoid it. GitLab itself is using Cloudflare for its CDN services and can handle thrown on it, but switching to https://statically.io as a full-fledged and free CDN intended as a CORS proxy would put unnecessary strain on it. The JupyterLite docs would have much less traffic in comparison.